enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -35,53 +35,70 @@ The purpose of log forwarding is to provide a declarative way by which adopters
 container and node logs to destinations that are not necessarily managed by the OKD cluster logging infrastructure.
 Destinations are either on or off cluster endpoints such as the cluster logging provided Elasticsearch, an organization's Kafka message bus, or other log aggregation solutions (e.g. Splunk, Graylog).
 
+This document describes the initial release goals for log forwarding. "Future plans" at the end outlines stories that were pushed out during discussions.
+
 ## Motivation
 
 Organizations desire to reuse their existing enterprise log solutions to store container logs.  Providing a declarative mechanism by which administrators define a log destination simplifies their operational burden.  They are able to take advantage of log collection infrastructure with minimal configuration changes. The cluster logging stack is able to deploy a collector to each OKD node that is configured with the necessary permissions to collect the logs, add container metadata (e.g. labels) and ship them to the specified endpoint.
 
-
-
 ### Goals
 The specific goals of this proposal are:
 
-* Provide an API that identifies log sources and their desired forwarding outputs.
-* Send logs to an Elasticsearch cluster not managed by the cluster logging infrastructure
-* Send logs to a fluentd instance not managed by the cluster logging infrastructure via fluentd's `forward` protocol
-* Allow deployment of log forwarding without deploying the entirety of the cluster logging infrastructure (e.g. Kibana, Elasticsearch)
-* Support TLS between the collector and destination if so configured
-* Use the same log forwarding logic to write logs to a specified output or to the default storage managed by cluster logging (e.g. Elasticsearch).
+* Selectively forward Application, Infrastructure and Audit inputs.
+* Forward any combination of inputs to any combination of outputs.
+* Send logs to endpoints not managed by the cluster logging infrastructure such as:
+  * An Elasticsearch cluster (version 5 or 6)
+  * A Greylog deployment
```

Comment:
nit: they call themselves Graylog

Reply:
Yes, of course they do. Like Tarzan the Earl of Greystoke. I remeber that episode when he tried to return home to England in a crate, but the folks in the village of Graystoke had it forwarded to Greystoke Alabama. That was a funny one.

Reply:
Of course they do. And they pronounce aluminium alooooomnm. But we must be tolerant and treat their software just we would any other software.